### DISCUSSION TITLE
separate jsx/tsx from js/ts

### DISCUSSION THREAD

**Discussion by developomp** [lowlighter/metrics#893] [Ideas]
Instead of putting jsx and tsx codes with js and ts in the language metrics, separate them into different categories.

**Comment by Nixinova** [comment]
linguist treats JSX and JS as the same, and puts TSX as a subcategory of TS. so TSX could be split if wanted but JSX cannot.

**Comment by spenserblack** [comment]
There's been a lot of discussion of how to handle jsx vs js in github/linguist, but https://github.com/github/linguist/pull/3677#issuecomment-440536127 summarizes well why they're considered the same language.
